import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        //expection handling try-catch
        /*try and catch we use for exception
        suppose we store array of 3 and we finding the value of 4th array
        that should we try the exception and catch
         */
        int[] marks ={97,98,95};
        try{
            System.out.println(marks[5]);
        }catch(Exception Exception){
            //do something after catching
        }




    }
}